cli/formatter/container.go

```diff
@@ -0,0 +1,56 @@
+package formatter
+
+import (
+	"github.com/alibaba/pouch/apis/types"
+	"github.com/alibaba/pouch/pkg/utils"
+)
+
+//ContainerHeader is the map to show container head
+type ContainerHeader map[string]string
+
+//ContainerContext is the map to show container context detail
+type ContainerContext map[string]string
+
+//NewContainerHeader is to generate a ContainerHeader to be show
+func NewContainerHeader() ContainerHeader {
```

Comment:
seem we define a const map here is enough, no need to define a func?

Reply:
yes , you are right. will change the func to const